starfish/test/full_pipelines/api/test_iss_api.py

```diff
@@ -123,6 +123,7 @@ def test_iss_pipeline_cropped_data(tmpdir):
 
     assert pipeline_log[0]['method'] == 'WhiteTophat'
     assert pipeline_log[1]['method'] == 'Warp'
+    assert 'transforms_list' in pipeline_log[1]['method_runtime_parameters']
```

Comment:
So when it logs the transforms list, what gets logged here?  The actual object?

Reply:
the string representation of the object